.github/workflows/zcash-lightwalletd.yml

```diff
@@ -11,7 +11,7 @@ on:
       # rebuild lightwalletd whenever the related Zebra code changes
       #
       # TODO: this code isn't compiled in this docker image
-      #       rebuild whenever the actual code at zcash/lightwalletd/master changes
+      #       rebuild whenever the actual code at lightwalletd/master changes
```

Comment:
```suggestion
      #       rebuild whenever the actual code at adityapk00/lightwalletd/master changes
```

Reply:
My idea was to avoid having unneeded repetition so that we don't need to change it if we ever change the repository again.

Reply:
I'll mark this as solved in order to fix CI since it's blocking other PRs. @dconnolly let me know if you'd like to really make this change, I can make it in another PR